### ISSUE TITLE
Update tooltip text for all social media tooltips

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/food-oasis#2272]
### Dependency
- [x] #2130

### Overview
Tooltips should provide helpful information to the user entering information in a form field. In this issue we will update tooltip text to make it more clear.

**Note** We will be using full URLs. This [comment](https://github.com/hackforla/food-oasis/issues/2128#issuecomment-2478781886) provides the latest as discussed by the team. 

### Action Items

- [x] Change tooltip text for Instagram, Facebook, LinkedIn, Pinterest, Twitter(X). For example as of Oct 28, 2025: Instead of saying ```Enter a full Instagram URL, e.g., https://www.instagram.com/hackforla```, it should say ```Enter organization's full Instagram URL, e.g., https://www.instagram.com/hackforla```. 
- [ ] Once the PR associated with this issue is merged or this issue is closed, check issue #2128 and mark the corresponding action item complete. Further, if all the action items in that issue are marked complete, move that issue to the new issue approval column so someone (or you) can check if that issue is complete and mark it done or reprioritize it if needs more work.

### Resources/Instructions
- Related issue #2128

**Comment by SAUMILDHANKAR** [comment]
I cleaned up the issue requirements. Now it is very tiny issue just to change some text and ready to be prioritized.
